PRIDE/donations.yml

```diff
@@ -36,4 +36,134 @@
   timestamp: 1618200920
   files:
     - PRIDE/2021-04-12.png
+ 
+- ticker: PRIDE
+  announcement_url: https://twitter.com/StakeWithPride/status/1384304031233757187
+  recipient:
+    name: AKT
+    contact: https://www.akt.org.uk/contact
+    tel:  "07467306483"
+    address: n/a
+  ada: 95
+  usd: 124
+  timestamp: 1618798164
+  files:
+    - PRIDE/2021-04-19.png
+
+- ticker: PRIDE
+  announcement_url: https://twitter.com/StakeWithPride/status/1385972401679175680
+  recipient:
+    name: OogaChaga
+    contact: https://oogachaga.com/contact
+    tel:  "8592 0609"
+    address: n/a
+  ada: 65.85
+  usd: 75.42
+  timestamp: 1619143764
+  files:
+    - PRIDE/2021-04-24.png
+
+- ticker: PRIDE
+  announcement_url: https://twitter.com/StakeWithPride/status/1386451918118023171
+  recipient:
+    name: Milaap
+    contact: https://milaap.org/contact-us
+    tel:  "8889688889"
+    address: n/a
+  ada: 88
+  usd: 95
+  timestamp: 1619316564
+  files:
+    - PRIDE/2021-04-25.png
+
+- ticker: PRIDE
+  announcement_url: https://twitter.com/StakeWithPride/status/1389994189195153409
+  recipient:
+    name: Sexual Minorities Uganda
+    contact: https://sexualminoritiesuganda.com/contact/
+    tel:  +256 39 2174432 
+    address: n/a
+  ada: 56.4
+  usd: 75
+  timestamp: 1620180564
+  files:
+    - PRIDE/2021-05-05.png
+
+- ticker: PRIDE
+  announcement_url: https://twitter.com/StakeWithPride/status/1395735815410511873
+  recipient:
+    name: Bisi Alimi Foundation
+    contact: https://www.bisialimifoundation.org/donate/
+    tel:  +234-8125549015
+    address: n/a
+  ada: 76
+  usd: 119.73
+  timestamp: 1621624087
+  files:
+    - PRIDE/prideMay21.png
+
+
+- ticker: PRIDE
+  announcement_url: https://twitter.com/StakeWithPride/status/1399048939454623751
+  recipient: MBARARA RISE FOUNDATION
```

Comment:
This is not valid against YAML conventions.
If you only want to use this recipient once you can just delete the name in this line.
If you want to reuse it you can give it a name like &mbaara and reuse it in later donations later with *mbaara
When reusing you will not need the sub-attributes (name, contact, ..)